client/graphics/passes/point-light-pass.hpp

```diff
@@ -0,0 +1,74 @@
+#pragma once
+
+#include "render-pass.hpp"
+#include "resources/obj.hpp"
+
+#include <memory>
+
+namespace tec::graphics::pass {
+using PointLightMap = Multiton<eid, PointLight*>;
+
+class PointLightPass final : public RenderPass {
+public:
+	PointLightPass() : RenderPass("PointLightPass") {
+		if (const auto sphere = OBJ::Create(Path("assets:/sphere/sphere.obj")); !sphere) {
+			spdlog::get("console_log")->debug("[RenderSystem] Error loading sphere.obj.");
+		}
+		else {
+			this->sphere_vbo.Load(sphere);
+		}
+	}
+	void Prepare(const GBuffer& gbuffer) override {
+		gbuffer.BeginLightPass();
+		gbuffer.BeginPointLightPass();
+	}
+	void Run(const gfx::ShaderSet& default_shaders, const View& view, const RenderItems&) override {
```

Comment:
I'm assuming the majority of the different graphics passes' `Run` functions are mostly copy-paste of the old functions, if this is not the case, I can give them a closer look, I didn't notice anything at a glance though.

Reply:
Yep, as best as I could, with some minor refactoring